.vscodeignore

```diff
@@ -7,5 +7,11 @@ src/**
 tsconfig.json
 vsc-extension-quickstart.md
 tslint.json
-org.jboss.tools.ssp.distribution-0.0.9-SNAPSHOT.zip
-*.vsix
\ No newline at end of file
+org.jboss.tools.rsp.distribution-*.zip
+*.vsix
+build/**
+coverage/**
+images/vscode-adapters.gif
```

Comment:
This file is used by README.md

Reply:
and it is also nearly 90 MB

Reply:
The readme gets pulled to marketplace directly from github, it doesn't even need to be in the vsix. And I really don't want the vsix to be ~120 MB.

Reply:
I don't agree check my extension https://marketplace.visualstudio.com/items?itemName=redhat.project-initializer it still displays Fabric8.... when the git repo has been updated to a new title

Reply:
But I agree if it's 90MB then we have an issue